### PR TITLE
[tflchef] Move cook and cook_graph into ModelChef

### PR DESCRIPTION
**Pull request by seanshpark**
This will revise ModelChef as to move cook and cook_graph methods into member.

### CODE REVIEW

File: compiler/tflchef/core/src/ModelChef.cpp [hunk 1]
```diff
@@ -201,7 +205,7 @@ struct ModelChef
   std::vector<flatbuffers::Offset<::tflite::SubGraph>> subgraph_vec;
   std::map<tflite::BuiltinOperator, int32_t> builtin_code_map;
   std::vector<std::string> custom_code_vec;
-  std::string graph_name;
+  std::string _graph_name;
```
Comment (seanshpark) [inline compiler/tflchef/core/src/ModelChef.cpp:208]: added `_` prefix to minimize changes of existing codes.

File: compiler/tflchef/core/src/ModelChef.cpp [hunk 2]
```diff
@@ -769,6 +950,9 @@ GeneratedModel cook(const ::tflchef::ModelRecipe &model_recipe)
   mc.flatbuffer_builder =
     std::unique_ptr<flatbuffers::FlatBufferBuilder>(new flatbuffers::FlatBufferBuilder(1024));
 
+  mc.cook(model_recipe);
+
+#if 0
```
Comment (seanshpark) [inline compiler/tflchef/core/src/ModelChef.cpp:955]: separated this block to `ModelChef::cook()` method.

**Comment by seanshpark** [comment]
- for #13627

**Review by mhs4670go — APPROVED** [review]
LGTM

**Pull request merged**